i18n/create_messages.py

```diff
@@ -34,15 +34,15 @@ def string_is_ascii(s):
     return True
   except UnicodeEncodeError:
     return False
-  
+
 def load_constants(filename):
   """Read in constants file, which must be output in every language."""
   constant_defs = read_json_file(filename);
   constants_text = '\n'
   for key in constant_defs:
     value = constant_defs[key]
     value = value.replace('"', '\\"')
-    constants_text += '\nBlockly.Msg.{0} = \"{1}\";'.format(key, value)  
```

Comment:
Wouldn't it be better to @export them?

The motivation to use @export is
1. no need to overwrite all existing code Blockly.Msg.SOMETHING to Blockly.Msg["SOMETHING"]
2. better optimization when used on many places (IMHO not very significant in this case)

See #1241 for more details.